test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -278,17 +280,18 @@ describe("BENEFICIAL OWNER GOV controller", () => {
       expect(resp.text).toContain(ErrorMessages.INVALID_DATE);
     });
 
-    test(`renders the current page ${config.BENEFICIAL_OWNER_GOV_PAGE} with DATE_NOT_IN_PAST error when start date is not in the past`, async () => {
-      const beneficialOwnerGov = REQ_BODY_BENEFICIAL_OWNER_GOV_FOR_DATE_VALIDATION;
-      beneficialOwnerGov["start_date-day"] =  "10";
-      beneficialOwnerGov["start_date-month"] = "10";
-      beneficialOwnerGov["start_date-year"] = "2050";
+    test(`renders the current page ${config.BENEFICIAL_OWNER_GOV_PAGE} with DATE_NOT_IN_PAST_OR_TODAY error when start date is not in the past`, async () => {
```

Comment:
could you add a check submitting todays date and making sure no error message is returned (just to test that today is allowed)

Reply:
i think we'll need that 'today' test in the other controllers changed too